src/components/LightUpChinatown/LightUpChinatownPage.tsx

```diff
@@ -12,22 +12,23 @@ import DonationSection from './DonationSection';
 import DonationDetail from './DonationDetail';
 import DonationProgressBar from './DonationProgressBar';
 import LightUpFaq from './LightUpFaq';
+import { getLightUpChinatownDonation } from '../../utilities/api';
 
 const LightUpChinatownPage = () => {
   const { t } = useTranslation();
   const today = new Date();
   const campaignEndDate = new Date('11/30/2020');
   const timeUntilEnd = campaignEndDate.getTime() - today.getTime();
   const daysUntilEnd = Math.ceil(timeUntilEnd / (1000 * 3600 * 24));
-  const [contributions, setContributions] = useState(0);
+  const [contributions, setContributions] = useState<number>(0);
+
+  const fetchData = async (project_id: number) => {
+    const { data } = await getLightUpChinatownDonation(project_id);
+    setContributions(data.amount_raised);
+  };
 
   useEffect(() => {
-    setContributions(15000);
-    // TODO: use the below after backend route to fetch raised is implemented
-    //
-    // const fetchData = async () => {
-    // setContributions(data.contributions.raised);
-    // };
+    fetchData(1);
```

Comment:
is this created in prod yet?

Reply:
what does "prod" mean?
I followed by this:
![Screen Shot 2020-11-13 at 11 51 59 PM](https://user-images.githubusercontent.com/47458946/99139961-6bcc9f00-260b-11eb-9e8d-f681e627aa96.png)
Is it correct?

Reply:
Mind adding it as a constant and importing it? `LIGHT_UP_CHINATOWN_PROJECT_ID`

Reply:
Okay!